challenges/exercises/ex45.py

```diff
@@ -0,0 +1,33 @@
+# Challenge 045
+
+
+"""
+Set the total to 0 to start with.
+While the total is 50 or less, ask the user to input a number.
+Add that number to the total and print the message "This is the [total]".
+Stop the loop when the total is over 500.
+"""
+
+
+def less_than_fifty(num):
+    total = 0
+    if isinstance(num, int):
+        while total <= 50:
```

Comment:
Here you could call the `user_input` function again to get a new num. You can then mock the function when testing (we can go over this if you come across this situation again).